app/js/views/reminders.jsx

```diff
@@ -117,6 +119,21 @@ export default class Reminders extends React.Component {
     this.setState({ reminders });
   }
 
+  onServiceworkerChange() {
+    if (this.speechController.idle) {
+      location.reload();
+    } else {
+      const message =
+        'The application has just been updated. '+
+        'You may experience random issues until the app is reloaded. ' +
+        `We'll reload the app automatically soon.`;
```

Comment:
With template strings you can have [multi line strings](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&code=const%20multilines%20%3D%20%60%0AHello%20%0AWorld!%0A%60%3B).

Maybe the copy here could be something a little less concerning -  'Project Cue is updating - please wait a moment'?